### PR TITLE
{devel}[GCCcore/6.3.0] texinfo v6.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2047.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2b4bb77c4d46cf07d69f8a623451958d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2462.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/eb74866316a22529708f2bb2f217b780 for a full test report.

File: easybuild/easyconfigs/t/texinfo/texinfo-6.4-GCCcore-6.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+
+builddependencies = [('binutils', '2.27')]
+
+osdependencies = ['texlive']
+
+preinstallopts = "make TEXMF=%(installdir)s/texmf install-tex && "
+
+# This will overwrite a users $TEXMFHOME so this module is best used as a build dependency
```
Comment (verdurin) [inline easybuild/easyconfigs/t/texinfo/texinfo-6.4-GCCcore-6.3.0.eb]: Should be `user's`

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca005.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/79b987935e26ac52332a729f3db05e07 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 11 (4 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1d218f0695943aaaf3ddf124b1e1c99e for a full test report.

**Comment by verdurin** [comment]
@boegel most of those failures are with old `Intel`-related toolchains and can be ignored, but the `tputs` ones might be genuine?

**Comment by boegel** [comment]
@verdurin Missing `ncurses` dep?

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 11 (4 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b3af10391f026ec8f593bb3263ebf6ce for a full test report.

**Comment by verdurin** [comment]
@boegel maybe, but in that case, why did the build succeed before?

**Comment by verdurin** [comment]
Still failing:
```
gcc  -g -O2   -o ginfo dir.o display.o dribble.o echo-area.o filesys.o footnotes.o gc.o indices.o info-utils.o info.o infodoc.o infomap.o m-x.o man.o nodemenu.o nodes.o search.o session.o signals.o terminal.o tilde.o variables.o window.o doc.o ../lib/libtxi.a ../gnulib/lib/libgnu.a   
terminal.o: In function `terminal_get_screen_size':
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:481: undefined reference to `tgetnum'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:493: undefined reference to `tgetnum'
terminal.o: In function `terminal_goto_xy':
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:235: undefined reference to `tgoto'
terminal.o: In function `terminal_delete_lines':
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:369: undefined reference to `tputs'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:365: undefined reference to `tgoto'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:365: undefined reference to `tputs'
terminal.o: In function `terminal_insert_lines':
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:393: undefined reference to `tputs'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:389: undefined reference to `tgoto'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:389: undefined reference to `tputs'
terminal.o: In function `terminal_initialize_terminal':
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:539: undefined reference to `tgetent'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:553: undefined reference to `tgetstr'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:579: undefined reference to `tgetstr'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:580: undefined reference to `tgetstr'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:581: undefined reference to `tgetstr'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:582: undefined reference to `tgetstr'
terminal.o:/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:585: more undefined references to `tgetstr' follow
terminal.o: In function `terminal_initialize_terminal':
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:617: undefined reference to `tgetflag'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:620: undefined reference to `tgetstr'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:628: undefined reference to `tgetstr'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:629: undefined reference to `tgetstr'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:630: undefined reference to `tgetstr'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:631: undefined reference to `tgetstr'
terminal.o:/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:633: more undefined references to `tgetstr' follow
terminal.o: In function `terminal_initialize_terminal':
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:617: undefined reference to `tgetflag'
terminal.o: In function `terminal_begin_using_terminal':
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:135: undefined reference to `tputs'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:144: undefined reference to `tputs'
terminal.o: In function `terminal_end_using_terminal':
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:166: undefined reference to `tputs'
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:175: undefined reference to `tputs'
terminal.o: In function `terminal_goto_xy':
/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:235: undefined reference to `tputs'
terminal.o:/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info/terminal.c:272: more undefined references to `tputs' follow
collect2: error: ld returned 1 exit status
make[3]: *** [Makefile:557: ginfo] Error 1
make[3]: Leaving directory '/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info'
make[2]: *** [Makefile:496: all] Error 2
make[2]: Leaving directory '/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13/info'
make[1]: *** [Makefile:516: all-recursive] Error 1
make[1]: Leaving directory '/dev/shm/texinfo/4.13a/dummy-/texinfo-4.13'
make: *** [Makefile:455: all] Error 2
```

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 19 (4 easyconfigs in this PR)
dell.lan - Linux fedora 26, Intel(R) Core(TM) i7-4600U CPU @ 2.10GHz, Python 2.7.14
See https://gist.github.com/af9e8f0fbbb0f0364723929b65ce1049 for a full test report.

**Comment by boegel** [comment]
No longer relevant, we have easyconfigs for newer version of `texinfo` already...

**Pull request closed without merging**